build.sbt

```
@@ -69,13 +69,35 @@ lazy val `lang-tests-js` = project
     testFrameworks += new TestFramework("utest.runner.Framework")
   )
 
-lazy val node = project.dependsOn(`lang-jvm`, `lang-testkit` % "test;test->test")
+lazy val node = project.dependsOn(`lang-jvm`)
 
-lazy val `grpc-server`    = project.dependsOn(node % "compile;test->test;runtime->provided")
-lazy val `ride-runner`    = project.dependsOn(node % "compile;test->test", `grpc-server`)
-lazy val `node-it`        = project.dependsOn(node % "compile;test->test", `lang-testkit`, `repl-jvm`, `grpc-server`)
-lazy val `node-generator` = project.dependsOn(node % "compile->test")
-lazy val benchmark        = project.dependsOn(node % "compile;test->test")
+lazy val `node-testkit` = project
+  .in(file("node/testkit"))
+  .dependsOn(`lang-jvm`, `lang-testkit` % "compile->compile", `node` % "compile->test")
```

1.  `node` already depends on `lang-jvm`.
2. `node` hasn't tests. So just `node` is enought.
3. Also:
   ```
   `lang-testkit` % "compile->compile"
   ```
   is the same as `lang-testkit`, see below.

So we can write:
```suggestion
  .dependsOn(`node`, `lang-testkit`)
```

---

Thank you, replaced.